R/dm.R

```diff
@@ -748,3 +748,54 @@ as.list.dm <- function(x, ...) { # for both dm and zoomed_dm
 as.list.zoomed_dm <- function(x, ...) {
   as.list(tbl_zoomed(x))
 }
+
+#' @export
+glimpse.dm <- function(x, width = NULL, ...) {
+  glimpse_width <- if (is.null(width)) {
+    getOption("width")
+  } else {
+    width
+  }
+  table_list <- dm_get_tables_impl(x)
+  if (length(table_list) == 0) {
+    cat_line(trim_width("dm of 0 tables", glimpse_width))
+    return(invisible(x))
+  }
+  cat_line(
+    trim_width(
+      paste0("dm of ", length(table_list), " tables: ", paste0(tick(names(table_list)), collapse = ", ")),
```

Comment:
If we go with commas I suggest the arg capped = TRUE here not to be redundant and display the total length twice.

```
commas(letters, capped = FALSE)
#> a, b, c, d, e, … (26 total)
commas(letters, capped = TRUE)
#> a, b, c, d, e, …
```

Reply:
I was thinking about `commas()`, but it seems to me that here it's not the best solution, since we don't know, how long the individual table names are, and the end result should have a max width, so what to choose for `MAX_COMMAS`? We could choose a really high number, or `glimpse_width`, but it seems a bit of an overkill. 
I like `toString()`, thanks for the tip.

Reply:
Using `toString()` now after 54c693d